_pytest/main.py

```diff
@@ -403,7 +403,7 @@ def _repr_failure_py(self, excinfo, style=None):
             else:
                 style = "long"
 
-        return excinfo.getrepr(funcargs=True,
+        return excinfo.getrepr(funcargs=True, abspath=True,
```

Comment:
I think this changes how paths are output to the user, right? If so, I think this PR should target `features` instead, because changing the output might break something (I have specially IDEs in mind, as I think PyDev relies on the current behavior).